cal/cal/static/css/calendaranalytics.css

```diff
@@ -10,22 +10,41 @@ h1, h2, h3, h4, h5, h6, a, p {
   color: #000; }
 
 .jumbotron h1, .jumbotron .h1 {
-  color: #000; }
+  color: #000;
```

Comment:
These changes should be made in the .scss file. Remember that .css is the compiled code, and will be erased as soon as the sass is recompiled.

Relevant command in cal/README.md